packages/react-experiments/src/components/MicroFeedback/MicroFeedback.types.ts

```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
 import { IComponent, IComponentStyles, ISlotProp, IStyleableComponentProps } from '@fluentui/foundation-legacy';
-import { ICalloutSlot, IListSlot } from '../../utilities/factoryComponents.types';
+import { IButtonSlot, ICalloutSlot, IListSlot } from '../../utilities/factoryComponents.types';
```

Comment:
For the microfeedback changes, we should look back at who contributed the component and reach out to them with a heads up about this change

Reply:
I'll do that as well.